src/components/NotesPanel/NotesPanel.scss

```diff
@@ -14,12 +14,14 @@
     display: flex;
     justify-content: space-between;
 
+    margin: 0 16px;
```

Comment:
added margin so that these 2 are aligned:
![image](https://user-images.githubusercontent.com/54084743/75900006-1f309300-5df1-11ea-8676-55310d2fd642.png)